src/Client.php

```diff
@@ -225,12 +225,25 @@ private function prepareEvent(Event $event, ?EventHint $hint = null, ?Scope $sco
 
         $this->addMissingStacktraceToEvent($event);
 
-        $event->setSdkIdentifier($this->sdkIdentifier);
-        $event->setSdkVersion($this->sdkVersion);
-        $event->setServerName($this->options->getServerName());
-        $event->setRelease($this->options->getRelease());
-        $event->setTags($this->options->getTags());
-        $event->setEnvironment($this->options->getEnvironment());
+        // Do not overwrite event data if it's already set
+        if (null !== $event->getSdkIdentifier()) {
```

Comment:
I think you meant to write `null === $event->...`, and the same thing for all the rest of the `if` conditions

Reply:
Whoops, sorry. Shouldn't write code after a busy day at work :)